src/components/providers/PostHogErrorBoundary.tsx

```
@@ -0,0 +1,87 @@
+            <div className="max-w-md text-center">
+              <h1 className="mb-4 text-2xl font-bold text-red-600">Something went wrong</h1>
+              <p className="mb-6 text-gray-600">
+                We&apos;ve been notified and will look into it. Please try refreshing the page.
+              </p>
+              <button
+                onClick={() => {
+                  this.setState({ hasError: false, error: undefined })
+                  window.location.reload()
+                }}
+                className="rounded-lg bg-blue-600 px-4 py-2 text-white hover:bg-blue-700"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)